src/intl/en.json

```diff
@@ -11,15 +11,15 @@
   "print": "Print",
   "this-form": "this form",
   "suggest-resource": "Suggest a resource",
-  "suggest-resource-form-id": "shr1CGrWlZldjacdE",
+  "suggest-resource-form-id": "shr2ARbWxFCI4xfQu",
```

Comment:
I don't have a solution for this yet, still digesting exactly how the forms are created, but there is hopefully a way to make spinning this off easier than someone having to manually paste them in + if there is a branch of the project meant for other to fork these should probably be empty

One idea is to add them to the .env file